conf/application.conf

```diff
@@ -31,6 +31,8 @@ play.http.filters = play.api.http.EnabledFilters
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "config.Module"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
Now that we're using `HttpClientV2Module` can you remove the deprecated `HttpClientModule` just above. This will cause a few errors as the connectors will also need updating to use new HttpClientV2 class. See readme here for migration guide https://github.com/hmrc/http-verbs